contracts/dev-utils/contracts/src/AssetBalance.sol

```diff
@@ -0,0 +1,374 @@
+/*
```

Comment:
Did any of the logic change between the code in contracts/src/LibAssetData.sol and this, or just refactored?

Reply:
Mostly refactored. `_getConvertibleMakerBalanceAndAssetProxyAllowance()` got added but it's just a no-op placeholder in this PR, so you'll see it again.